HousingRepairsOnlineApi/Helpers/AppointmentTimeToRepairAvailabilityMapper.cs

```diff
@@ -13,7 +14,7 @@ public RepairAvailability Map(AppointmentTime appointmentTime)
         {
             StartDateTime = appointmentTime.StartTime,
             EndDateTime = appointmentTime.EndTime,
-            Display = $"{appointmentTime.StartTime.Date.ToString("dd MMMM yyyy")} between {appointmentTime.StartTime.ToString("h:mmtt")} to {appointmentTime.EndTime.ToString("h:mmtt")}",
+            Display = $"{appointmentTime.StartTime.Date.ToString("dd MMMM yyyy", CultureInfo.InvariantCulture)} between {appointmentTime.StartTime.ToString("h:mmtt", CultureInfo.InvariantCulture)} to {appointmentTime.EndTime.ToString("h:mmtt", CultureInfo.InvariantCulture)}",
```

Comment:
Did you intend to rollback these changes too?

Reply:
Yes. I have reverted.